devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -205,17 +201,18 @@ def configure() {
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
-	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
-	refresh()
+	zigbee.setColorTemperature(7000) +
```

Comment:
I intended the 7000 just as a test but thought it would be best to leave it at 6500 for the release. Unless you think 7000 is better?

Reply:
@tpmanley You are right, let's leave 6500K here (and we will test setting 7000K on a side).